src/pages/Gateway/index.tsx

```diff
@@ -317,9 +342,9 @@ const Gateway = () => {
         label: changed.label as string,
         minDelegatedStake:
           formState.allowDelegatedStaking && changed.minDelegatedStake
-            ? new IOToken(
-                parseFloat(changed.minDelegatedStake as string),
-              ).toMIO().valueOf()
+            ? new IOToken(parseFloat(changed.minDelegatedStake as string))
+                .toMIO()
+                .valueOf()
```

Comment:
Is this supposed to display as mio? seems odd from a ux point of view

Reply:
I think it's odd but correct, as it's read from contract and written to contract as mIO. The user views and edits in IO. This part of the code is updating the AoUpdateGatewaySettingsParam so is converting to mIO.